src/maker.rs

```diff
@@ -57,7 +57,23 @@ impl Maker {
         }
     }
 
-    pub fn update_rate(&mut self, mid_market_rate: ()) {}
+    pub fn update_rate(
+        &mut self,
+        mid_market_rate: MidMarketRate,
```

Comment:
:question: Should we check that this rate is newer than the one we have? I don't think this function can trust that the argument is the latest mid market rate, as it doesn't know where it comes from.

Reply:
This check would not have much value given the current implementation - and the Kraken API. The ticker actually does not give us a timestamp. We could give it a timestamp inside `get_btc_dai_mid_market_rate` but there is not much value in that. 
Thus I removed the complete timestamp check for now. Whenever this function is called the maker _always_ assumes it is a new one.

Reply:
True, this was a comment on the first commit.